src/app/[lang]/(protected)/create-new-password/newPassPut.ts

```diff
@@ -0,0 +1,35 @@
+"use server";
+
+import { cookies } from "next/headers";
+import { redirect } from "next/navigation";
+import { env } from "@/env.mjs";
+import { getJwtAccessToken } from "../regenerateAccessToken";
+
+export async function newPassPut(
+  data: string,
+  lang: "en" | "uk",
+): Promise<string> {
+
+  const jwtAccessToken = await getJwtAccessToken();
+
+  if (!jwtAccessToken) {
+    redirect(`/${lang}`);
+  }
+
+  const response = await fetch(`${env.SERVER_URL}/api/user/new-password/save`, {
+    body: JSON.stringify({ userPassword: data }),
+    method: "PUT",
+    headers: {
+      Authorization: `Bearer ${jwtAccessToken}`,
+      "Content-Type": "application/json",
+    },
+  });
```

Comment:
In the Ukrainian interface, if I enter the same password as the previous one, I receive an error message in the English language. It should be in the Ukrainian